website/assets/js/pages/customers/new-license.page.js

```diff
@@ -0,0 +1,101 @@
+parasails.registerPage('new-license', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    // Form data
+    formData: { /* … */ },
+
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    quoteFormRules: {
+      numberOfHosts: {required: true},
+    },
+
+    billingFormRules: {
+      paymentSource: {required: true},
+    },
+
+    // Syncing / loading state
+    syncing: false,
+
+    // Server error state
+    cloudError: '',
+
+    // Success state when form has been submitted
+    cloudSuccess: false,
+    showBillingForm: false,
+    showQuotedPrice: false,
+    quotedPrice: undefined,
+    numberOfHostsQuoted: undefined,
+    thirtyDaysFromTodayInMS: Date.now() + 30*24*60*60*1000,
+    calendlyLink: '',
+    orderComplete: false,
+  },
+
+  //  ╦  ╦╔═╗╔═╗╔═╗╦ ╦╔═╗╦  ╔═╗
+  //  ║  ║╠╣ ║╣ ║  ╚╦╝║  ║  ║╣
+  //  ╩═╝╩╚  ╚═╝╚═╝ ╩ ╚═╝╩═╝╚═╝
+  beforeMount: function() {
+    let today = new Date(Date.now());
+    this.calendlyLink = 'https://calendly.com/fleetdm/demo?month='+today.getFullYear()+'-'+today.getMonth();
+  },
+  mounted: async function() {
+    //…
+
+  },
+
+  //  ╦╔╗╔╔╦╗╔═╗╦═╗╔═╗╔═╗╔╦╗╦╔═╗╔╗╔╔═╗
+  //  ║║║║ ║ ║╣ ╠╦╝╠═╣║   ║ ║║ ║║║║╚═╗
+  //  ╩╝╚╝ ╩ ╚═╝╩╚═╩ ╩╚═╝ ╩ ╩╚═╝╝╚╝╚═╝
+  methods: {
+
+    submittedPaymentForm: async function() {
+      // After payment is submitted, Display a success message and let them navigate to the dashboard
+      this.syncing = true;
+      this.orderComplete = true;
+      await this.forceRender();
+      this.$focus('[purpose="submit-button"]');
+      this.syncing = false;
```

Comment:
I love that you saw this `await` and protected it by working with `this.syncing`.

But since `await this.forceRender()` will never take more than 1ms (it's just waiting for Vue's virtual DOM to repaint), you can leave out the code that sets `this.syncing` here.